src/index.js

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
+import { Provider } from "react-redux";
 import App from "./components/App/App";
 import ErrorBoundary from "./components/ErrorBoundary";
+import createStore from "./reducers";
+
+const store = createStore();
 
 ReactDOM.render(
   <React.StrictMode>
     <ErrorBoundary>
-      <App />
+      <Provider store={store}>
+        <App />
+      </Provider>
     </ErrorBoundary>
```

Comment:
Not a problem or request, but as we talk about boilerplate, if you wanted to, all your additions here could totally be put in some other file:

```js
// provider.js
import { Provider } from "react-redux";
import createStore from "./reducers";

const store = createStore();

export const ReduxProvider = ({ children }) => {
  return (
    <Provider store={store}>
      {children}
    </Provider>
  );
}
```

If you did, in this file you'd need simply to import and then wrap `<App />` in `<ReduxProvider>`.

Does it save anything? No. But if you added more complex logic in redux, like say persisting your redux store in local storage, the file might end up being more redux than app. This encapsulation can help, but might also be overengineering now :)

Reply:
Awesome, thanks! I'll keep it in mind.